### PR TITLE
docs(ai): add new AI SDK documentation

### PR DESCRIPTION
**Pull request by rickstaa**
This pull request adds documentation for the new AI SDKs.

### CODE REVIEW

File: ai/builders/gateways.mdx [hunk 1]
```diff
@@ -0,0 +1,723 @@
...
+use to interact with the Livepeer AI network. Each gateway has its own unique
+features and capabilities. Here are some of the gateways that you can use to
+interact with the Livepeer AI network.
+
+<CardGroup cols={2}>
+  <Card
+    title="Livepeer Cloud Community Gateway"
+    href="https://www.livepeer.cloud/"
+    icon={
+      <svg
```
Comment (rickstaa) [inline ai/builders/gateways.mdx:15]: @victorges sadly mintlify doesn't yet accept svg paths. I submitted a feature request with the Mintlify team (see https://mintlify-users.slack.com/archives/C05D00NS55F/p1726563283058219?thread_ts=1726305209.436119&cid=C05D00NS55F).

File: ai/builders/gateways.mdx [hunk 2]
```diff
@@ -0,0 +1,723 @@
+---
+title: AI Gateways
+---
+
+There are multiple AI gateways available in the Livepeer ecosystem that you can
```
Comment (rickstaa) [inline ai/builders/gateways.mdx:5]: @hthillman Can you check that the studio branding is to your liking?

**Comment by rickstaa** [comment]
@victorges I noticed that the message on the `/ai/buidlers/get-started` needs to be changed will do that tomorrow 👍🏻. You could review the other pages, otherwise I will ping you tomorrow.

**Review by victorges — APPROVED** [review]
LGTM

**Comment by victorges** [comment]
FYI just merged an automatic update to the docs that seemed to have conflicted with some of your changes (e.g. deleting gateway.openapi.yaml, which seems to be recreated by speaskeasy)

**Pull request merged**